processor/sumologicprocessor/config.go

```diff
@@ -0,0 +1,79 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package sumologicprocessor // import "github.com/open-telemetry/opentelemetry-collector-contrib/processor/sumologicprocessor"
+
+import (
+	"go.opentelemetry.io/collector/component"
+)
+
+type Config struct {
+	AddCloudNamespace           bool                      `mapstructure:"add_cloud_namespace"`
+	TranslateAttributes         bool                      `mapstructure:"translate_attributes"`
+	TranslateTelegrafAttributes bool                      `mapstructure:"translate_telegraf_attributes"`
+	NestAttributes              *NestingProcessorConfig   `mapstructure:"nest_attributes"`
+	AggregateAttributes         []aggregationPair         `mapstructure:"aggregate_attributes"`
+	LogFieldsAttributes         *logFieldAttributesConfig `mapstructure:"field_attributes"`
+	TranslateDockerMetrics      bool                      `mapstructure:"translate_docker_metrics"`
+}
+
+type aggregationPair struct {
+	Attribute string   `mapstructure:"attribute"`
+	Patterns  []string `mapstructure:"prefixes"`
```

Comment:
Should the field name and config key name match more closely? "Patterns" and "prefixes" seem close but still distinct to me.

Reply:
I unified them to prefixes